shared/actions/json/fs.json

```diff
@@ -26,11 +26,12 @@
       "public": "I.Map<string, Types.Tlf>",
       "team": "I.Map<string, Types.Tlf>"
     },
-    "filePreviewLoad": {
+    "pathItemLoad": {
       "path": "Types.Path",
-      "identifyBehavior?": "?RPCTypes.TLFIdentifyBehavior"
+      "identifyBehavior?": "?RPCTypes.TLFIdentifyBehavior",
+      "silenceErrors?": "?boolean"
```

Comment:
This ends up not needed in this PR as we actually don't need this in widget, but since I already did it, I'm leaving it here in case we need it later.